packages/terra-icon/package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "csvtojson": "^2.0.8",
     "jsdom": "^15.1.1",
-    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/v1.41.2.tar.gz",
+    "one-cerner-style-icons": "https://github.com/cerner/one-cerner-style-icons/archive/v1.45.0.tar.gz",
```

Comment:
The PR says you're upgrading to 1.44, but here it is 1.45.

Reply:
I've changed the title. 1.45 is correct.